azDeploySecureSub.bicep

```diff
@@ -54,6 +71,8 @@ var planName = nameModule[5].outputs.shortName
 var uamiName = nameModule[6].outputs.shortName
 var dplscrName = nameModule[7].outputs.shortName
 var lawName = nameModule[8].outputs.shortName
+var avdName = nameModule[9].outputs.shortName
+var customRdpProperty = 'audiocapturemode:i:1;camerastoredirect:s:*;audiomode:i:0;drivestoredirect:s:;redirectclipboard:i:1;redirectcomports:i:0;redirectprinters:i:1;redirectsmartcards:i:1;screen mode id:i:2;devicestoredirect:s:*'
```

Comment:
If we're not making this a parameter, I am not sure this should go in the main bicep file.